themes/ddbasic/README.md

```diff
@@ -1,18 +1,206 @@
+
+Hiding a few fields with CSS is preferred to increasing the number of view modes. 
+
+Rendered view modes is the preferred method of presentation of entities as opposed to fielded views. 
+
+**Rendering ting materials**
+
+A recurring frontend task is to present a list of materials. The prefered procedure is to render an array of material ids as a carousel:
+
+```php
+theme(‘ting_carousel’, array( ‘items’ => array(id1,id2,id3) ) );
+```
+
+The carousel can hold alot of materials and still keep the UI clean across devices. 
+
+Future versions of the carousel could include custom settings for the user to render the materials more compact, with more deltails or as lists.
+
+### Field type ting_reference
+
+Any entity can reference specific ting materials via the ting_reference field type which in turn is easy to edit for the editors. Reuse this field when possible instead of introducing new fields for referencing materials.
+
+Render this field with the desired viewmode to output a list of materials from an entity which can be edited in the CMS.
+
+## Image styles
+
+Drupal image styles makes preprocessing and cropping of images easy. Fluid responsive design and everchanging DPIs of devices makes it unfeasible to render images in 1:1 with the display on each device. The purpose left for the image styles is to deliver relevant aspect ratios and *reasonable* resolution of the content.
+
+Upscaling should be avoided in all cases, and moderate CCS downscaling and optionally cropping is the prefered way of delivery for images. Always try to find a proper existing image style optionally downscaling it instead of introducing a new imagestyle with perfect pixel dimensions.
+
+### Manual crop
+
+[Manual cropping](https://www.drupal.org/project/manualcrop) is used to create optional crops of important aspect ratios. 
+
+* Keep the number of manual crop instances minimal
+* There must never be two manual crop instances with the same aspect ratio.
+* Always derive new image styles from manual crops in order to be able to reuse the manual crop in other styles
+* Design for existing manual crop aspect ratios
+* If no aspect matches a desired manual crop, consider using a manual crop very close to the aspect and crop in a derived style or in CSS
+
+## HTML markup
+
+Along with preprocessing done in template.php the [template files (temp link)](https://github.com/b14/ding2ddbasic/tree/master/themes/ddbasic/templates) are the html implementation of the Drupal objects. 
+
+DDBasic follows [Drupal best practices](https://www.drupal.org/coding-standards/css/architecture) for frontend development and Drupals take on CSS and best practices for SCSS.
+
+DDBasic introduces reusable html classes of special purpose which are documented in KSS along with their corresponding SCSS implementation. [See KSS Styleguide (temp link)](http://msd.ding2.server003.b14cms.dk/profiles/ding2/themes/ddbasic/styleguide)
```

Comment:
Agree. Let's have a seperate issue for compiled doc pages and related discussions re CI and compliance.
Let me know if core or DDB have a temporary URL instead of the msd URLs for this PR.

Reply:
:+1: [I created an issue for this](https://platform.dandigbib.org/issues/3183).

We will have to let links be until that is fixed.